build.gradle

```diff
@@ -295,6 +295,7 @@ dependencies {
     }
     implementation "org.springframework.boot:spring-boot-starter-tomcat:${spring_boot_version}"
     implementation "org.springframework.boot:spring-boot-starter-websocket:${spring_boot_version}"
+    implementation "org.springframework.boot:spring-boot-starter-artemis:${spring_boot_version}"
```

Comment:
Do we need the artemis dependency because previously we did not forward websocket mesages to any other instances? (I'm not really familiar with our setup for multiple instances.

Reply:
It is a bit confusing :D
Artemis (the learning platform) uses [Apache's ActiveMQ Artemis](https://activemq.apache.org/components/artemis/) (a message broker) to distribute messages between different instances of the learning platform.

We previously only used functionality Spring provided within `org.springframework.boot:spring-boot-starter-websocket`, but now also require additional functionality to parse JMS messages that were relayed between Artemis (learning platform) instances.

So for the automatic forwarding of messages from one instance of the learning platform to the instance to which an user is connected, we only used logic provided directly by Spring.
For forwarding messages from the user to the correct instance of the learning platform, we require the `org.springframework.boot:spring-boot-starter-artemis`-dependency.

Does that answer your question?

Reply:
Yes it does. :)